packages/angular_devkit/build_ng_packagr/src/build/schema.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Considering the draft won't be changing much, we should start using a non-draft version of the schema spec; `http://json-schema.org/schema`.

Reply:
`http://json-schema.org/schema` points to latest version of the schema and will change over time.  It currently points to the `draft-07` version which allows everything to work for now.  However, there is no guarantee that the latest version will be supported by the CLI nor that the file will conform to the latest version.  By specifying a specific version the `$schema` identifier can be used as both a version identifier for the format of the file and as the location of the schema.  This allows processors to use the specified version/format of the specification when parsing and processing the file.  (ref: http://json-schema.org/draft-07/json-schema-core.html#rfc.section.7)

Reply:
(we use that value too when running quicktype)

This is non blocking, but we use the "latest" URI in all but 2 of our schemas, so I'd say it's better to use the same everywhere than just keep splitting the code base.